app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/viewmodel/PledgedProjectsOverviewViewModel.kt

```diff
@@ -34,6 +34,12 @@ class PledgedProjectsOverviewViewModel(environment: Environment) : ViewModel() {
     val ppoCardsState: StateFlow<PagingData<PPOCardDataMock>> = ppoCards.asStateFlow()
     val totalAlertsState: StateFlow<Int> = totalAlerts.asStateFlow()
 
+    fun showSnackbarAndRefreshCardsList() {
+        snackbarMessage.invoke(R.string.Confirm) // TODO: Replace with actual translated string
```

Comment:
Until the strings are translated, we are adding the strings with "FPO" in the title. You can see an example and add the string here at the end of this file:  https://github.com/kickstarter/android-oss/blob/207fb925d80b37291a44126c964d0c5e32f3b8f1/app/src/main/res/values/strings.xml#L91